addons/handhelds/XEH_PREP.sqf

```diff
@@ -0,0 +1,3 @@
+
+[QPATHTOF(functions\DOUBLES(fnc,updateProgrammatorDialog).sqf), QUOTE(DFUNC(updateProgrammatorDialog))] call CBA_fnc_compileFunction;
```

Comment:
I guess prep would do TFAR_fnc_... ? without the module name?

Reply:
yes, but since it's a new function and specifically for handhelds.
It's just the macros demacroized